### PR TITLE
Arbitrary test fixture

### PR DESCRIPTION
**Pull request by paul-m**
Allow for placing the fixture in an arbitrary location. This will help later when we move to sites/simpletest/ or whatever it ends up being.

### CODE REVIEW

File: tests/src/Functional/ScaffoldTest.php [hunk 1]
```diff
@@ -101,7 +96,7 @@ public function scaffoldFixturesThatThrowTestValues() {
     return [
       [
         'drupal-drupal-missing-scaffold-file',
-        'assertDrupalDrupalSutWasScaffolded',
+        '_no_assertion_',
```
Comment (greg-1-anderson) [inline tests/src/Functional/ScaffoldTest.php:99]: Why did you make this `_no_assertion_`? The test was passing here before, wasn't it?

**Comment by greg-1-anderson** [comment]
Awesome 👍 🚢 🚀

File: tests/src/Functional/ScaffoldTest.php (hunk 1 above)
Comment (paul-m) [reply tests/src/Functional/ScaffoldTest.php:99]: Yes, but it doesn't actually make an assertion.

Comment (greg-1-anderson) [reply tests/src/Functional/ScaffoldTest.php:99]: Ah, right, this data provider should just remove its second parameter, as its unused.

**Comment by greg-1-anderson** [comment]
@grasmash If you merge #16 first, GitHub should also mark this PR as merged. I combined @paul-m 's changes in my PR.

I fixed the 'no assertions' problem in 218ed7b.

**Comment by greg-1-anderson** [comment]
GitHub will probably figure things out if you merge here first, too. I might not have time to fix my test error for a bit.

**Pull request merged**